src/components/Sing/ScoreSequencer.vue

```diff
@@ -992,22 +996,37 @@ export default defineComponent({
       }
     };
 
-    onMounted(() => {
+    onActivated(() => {
       const sequencerBodyElement = sequencerBody.value;
       if (!sequencerBodyElement) {
         throw new Error("sequencerBodyElement is null.");
       }
-      // 上から2/3の位置がC4になるようにスクロールする
-      const clientHeight = sequencerBodyElement.clientHeight;
-      const c4BaseY = noteNumberToBaseY(60);
-      const clientBaseHeight = clientHeight / zoomY.value;
-      const scrollBaseY = c4BaseY - clientBaseHeight * (2 / 3);
-      sequencerBodyElement.scrollTo(0, scrollBaseY * zoomY.value);
 
-      // スクロールバーの幅を取得する
-      const clientWidth = sequencerBodyElement.clientWidth;
-      const offsetWidth = sequencerBodyElement.offsetWidth;
-      scrollBarWidth.value = offsetWidth - clientWidth;
+      let xToScroll = 0;
+      let yToScroll = 0;
```

Comment:
こうも書けるかもです！
（以降の処理で初期化していなかったときにTypeScriptの静的解析でたぶん気付ける＆0初期化は意味ないということがわかる）
```suggestion
      let xToScroll: number;
      let yToScroll: number;
```